task/buildah/0.6/buildah.yaml

```diff
@@ -132,6 +132,21 @@ spec:
       Note: the SBOM from the prefetch task - if there is one - must be in the same format.
     type: string
     default: spdx
+  - name: SBOM_SYFT_SELECT_CATALOGERS
+    description: >-
+      Extra option to customize Syft's default catalogers when generating SBOMs.
+      The value corresponds to Syft's CLI flag --select-catalogers. The details
+      about available catalogers can be found here:
+      https://github.com/anchore/syft/wiki/Package-Cataloger-Selection
+    type: string
+    default: ""
+  - name: SBOM_SOURCE_SCAN_ENABLED
+    description: >-
+      Flag to enable or disable SBOM generation for source code. By default the scanner
+      of the source code is enabled and can be disabled if the SBOM_SYFT_SELECT_CATALOGERS
+      can't turn off catalogers that cause false positives on source code scanning.
```

Comment:
Let's mention that source scan is only enabled for non-hermetic builds - `SBOM_SOURCE_SCAN_ENABLED=true + HERMETIC=true` skips the source scan